### PR TITLE
Add deprecation notice for system.swap on windows

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Adds deprecation notice to releasenotes.

### CODE REVIEW

File: releasenotes/notes/deprecate-win-system.swap-2653c98a50f42ba2.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    On Windows, the metrics `system.swap.*` are being deprecated.  They have
```
Comment (olivielpeau) [inline releasenotes/notes/deprecate-win-system.swap-2653c98a50f42ba2.yaml]: we could be a bit more explicit by saying that they'll be removed in a future major version of the Agent, what do you think? For example:

```
    On Windows, the metrics `system.swap.*` are being deprecated and will be removed in a future major version of the Agent.
```

File: releasenotes/notes/deprecate-win-system.swap-2653c98a50f42ba2.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    On Windows, the metrics `system.swap.*` are being deprecated and will be
+    removed in a future major version of the Agent.  They have been replaced
```
Comment (KSerrania) [inline releasenotes/notes/deprecate-win-system.swap-2653c98a50f42ba2.yaml]: Nit: there's one space too many after the period

**Review by albertvaka — APPROVED** [review]

**Comment by olivielpeau** [comment]
our plans have changed: we'll re-introduce the `system.mem.swap.*` metrics on v6 because we've realized the `system.swap.*` in v6 were not exactly the same as v5's `system.mem.swap.*` metrics

Closing

**Comment by olivielpeau** [comment]
(See follow-up here: https://github.com/DataDog/datadog-agent/pull/3582)

**Pull request closed without merging**